geomesa-kafka/geomesa-kafka-confluent/src/main/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreFactory.scala

```diff
@@ -29,8 +34,11 @@ class ConfluentKafkaDataStoreFactory extends DataStoreFactorySpi {
   override def createDataStore(params: java.util.Map[String, Serializable]): KafkaDataStore = {
     val config = KafkaDataStoreFactory.buildConfig(params)
     val url = ConfluentKafkaDataStoreFactory.SchemaRegistryUrl.lookup(params)
+    val schemaOverridesConfig = ConfluentKafkaDataStoreFactory.SchemaOverrides.lookup(params)
+    val schemaOverrides = ConfluentKafkaDataStoreFactory.parseSchemaOverrides(schemaOverridesConfig)
```

Comment:
can we change this to be `lookupOpt` and then pass an option into `ConfluentKafkaDataStoreFactory.parseSchemaOverrides`? that way we're clear that it may be null